website/docs/reference/resource-properties/constraints.md

```diff
@@ -518,3 +518,60 @@ alter table schema_name.my_model add constraint 472394792387497234 check (id > 0
 </div>
 
 </WHCode>
+
+### Custom constraints on models for advanced configuration of tables
+
+In dbt Cloud, you can use custom constraints on models for advanced configuration of tables. For example, custom constraints allow you to set [masking policies](https://docs.snowflake.com/en/user-guide/security-column-intro#what-are-masking-policies) in Snowflake when using a Create Table As Select (CTAS).
+
+Each data warehouse has its own set of parameters that can be set for columns in their CTAS statements. For example [Databricks](https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-create-table-using.html) and [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#column_name_and_column_schema)".
+
```

Comment:
i think before introducing the tag way, should we say there are a couple of ways to do this:

the tag way (with a link to the tag way)
the non-tag way (with a link to that)

Reply:
Hiya @mirnawong1 

Thanks for this suggestion. Do you know which links would be best to add to the docs for the tag and non-tag ways?

Kind Regards
Natalie